TheWarWithin/RogueSubtlety.lua

```diff
@@ -1475,19 +1449,31 @@ spec:RegisterAbilities( {
     symbols_of_death = {
         id = 212283,
         cast = 0,
-        charges = function() if talent.death_perception.enabled then return 1 end end,
-        cooldown = 30,
+        charges = function() if talent.death_perception.enabled then
+            return talent.death_perception.rank + 1 else return nil end
```

Comment:
`else return nil` is not needed -- if the function ends without returning a value, it returns nil by default.

Reply:
Oops, misremembered the explanation from last time. So close.